pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs

```diff
@@ -0,0 +1,355 @@
+                </div>
+                <div class="radio">
+                  <label><input type="radio" name="optradio" value = 2>Grouping not-assigned students (fill existing groups first)</label>
+                </div>
+                <div class="radio">
+                  <label><input type="radio" name="optradio" value = 3>Grouping not-assigned students (will not change existing groups; create new groups only)</label>
+                </div>
+                <div class="d-flex justify-content-end">
+                  <div class="form-group mb-0">
+                    <input type="hidden" name="__action" value="auto_assessment_groups">
+                    <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+                    <button type="submit" class="btn btn-primary">Group</button>
+                  </div>
+                </div>
+              </form>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div id="content" class="container-fluid">
+        <div class="modal fade" id="copyAssessmentGroupsModal" tabindex="-1" role="dialog" aria-labelledby="copyAssessmentGroups">
+          <div class="modal-dialog" role="document">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h4 class="modal-title" id="copyAssessmentGroupsModalLabel">Copy group setting</h4>
+              </div>
+              <div class="modal-body">
+                <form name="copy-assessment-group-form" method="POST" enctype="multipart/form-data">
+                  <div class="input-group mb-3">
+                    <select class="custom-select" name="inputGroupSelect01">
+                      <option selected>Choose Assessment...</option>
+                      <% assessment_list_rows.forEach(function(row, iRow) {%>
+                        <option value = <%= row.id %> ><%= row.tid %>: <%= row.title %></option>
+                      <% }); %>
+                    </select>
+                  </div>
+                  <div class="d-flex justify-content-end">
+                    <div class="form-group mb-0">
+                      <input type="hidden" name="__action" value="copy_assessment_groups">
+                      <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                      <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+                      <button type="submit" class="btn btn-primary">Copy</button>
+                    </div>
+                  </div>
+                </form>
+              </div>
+            </div>
+          </div>
+        </div>
+      <div class="modal fade" id="addGroupModal" tabindex="-1" role="dialog" aria-labelledby="addGroup">
```

Comment:
```suggestion
      <div class="modal fade" id="addGroupModal" tabindex="-1" role="dialog" aria-labelledby="addGroupModalLabel">
```
This should match the `id` on line 118. `aria-labelledby` tells screen readers where to find the title of this div.